source/includes/markdown/_03-auth.html.md

```diff
@@ -356,7 +356,7 @@ curl https://app.asana.com/api/1.0/users/me \
   -H "Authorization: Bearer ACCESS_TOKEN"
 ```
 
-You can generate a personal access token from the Asana [developer console](https://app.asana.com/0/developer-console). See the [Authentication quick start](/docs/authentication-quick-start) for detailed instructions on getting started with PATs. 
+You can generate a personal access token from the Asana [developer console](https://app.asana.com/0/developer-console). See the [authenticating](/docs/authenticating) for detailed instructions on getting started with PATs. 
```

Comment:
Did you check if there are external resources linked to `/docs/authentication-quick-start`? This change will break external links to this resource

Reply:
Fixed!